wp-content/themes/core/integrations/gravity-forms/css/default.pcss

```diff
@@ -23,6 +23,33 @@
 			padding: 0;
 			font-size: 16px;
 		}
+
+		/* CASE: Lead Form: Form Fields Inline */
+		.b-lead-form--fields-inline & {
+			display: flex;
+			align-items: center;
+		}
+	}
+
+	.gform_body {
+		/* CASE: Lead Form: Form Fields Inline */
+		.b-lead-form--fields-inline & {
+			flex: 1;
+		}
+	}
+
+	.gfield_label {
+		/* CASE: Lead Form: Background Dark */
+		.b-lead-form--dark & {
+			color: var(--color-white);
+		}
+	}
+
+	.gform_footer {
+		/* CASE: Lead Form: Form Fields Inline */
+		.b-lead-form--fields-inline & {
+			margin-left: var(--spacer-10);
```

Comment:
With these styles being more about the block than the integration itself, I'm wondering if it would be more helpful to add these styles within the block css vs. here to keep the separation between block and integration cleaner?

Reply:
Previous block notes kinda had this type of setup where if you're modifying a class, make it a case and put it within that file. I could see situations where other blocks need an inline form or a white label on a black background. Maybe keep them here but make the class names more reusable like `.is-form-inline` ?

Reply:
Noting our follow up to dos:

- Will update the color/theme based classed to leverage existing SQ1 `t-theme--light` for consistency
- The inline layout helper class is really contextually centered on the GF integration and could be handy to have available outside of the context of the lead form block or any block for that matter. We'll turn it into a complex layout utility that is GF only

Thanks Scott!